projects/plugins/jetpack/extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -159,13 +166,13 @@ function render_player( $player_data, $attributes ) {
 	$player_classes_name  = trim( "{$secondary_colors['class']} {$background_colors['class']}" );
 	$player_inline_style  = trim( "{$secondary_colors['style']} ${background_colors['style']}" );
 	$player_inline_style .= get_css_vars( $attributes );
-
-	$block_classname = Blocks::classes( FEATURE_NAME, $attributes, array( 'is-default' ) );
-	$is_amp          = Blocks::is_amp_request();
+	$wrapper_attributes   = \WP_Block_Supports::get_instance()->apply_block_supports();
```

Comment:
Like we discussed in another PR, I think, if we're to use this in more and more places, it will start making more sense to refactor `Blocks::classes` to use `apply_block_supports` behind the scenes, so we can keep the extraction and avoid duplication. It makes even more sense with blocks like this one where we have the extra `is-default` class.